.fastlane/Fastfile

```diff
@@ -138,6 +138,12 @@ platform :ios do
       precheck_include_in_app_purchases: false
     )
 
+    upload_symbols_to_crashlytics(
+      gsp_path: "./Frameworks/native-secrets/ios/Firebase-Production/GoogleService-Info.plist",
+      binary_path: "./bin/upload-symbols",
+      dsym_path: "#{gym_dir}/Kickstarter.app.dSYM.zip"
```

Comment:
@singhhari I wonder if this removed line is why our `dSyms` aren't being uploaded anymore? Seems like 4.13.1 didn't get its `dSyms` uploaded.

Reply:
Actually only one of the dSyms failed to upload, so it might be a one-off. Let's keep an eye on it.